File: iniciante/1036/1036.cpp
// 1036 - Fórmula de Bhaskara - beecrowd
// Iniciante - Nível 3

#include <iostream>
#include <iomanip>
#include <cmath>

using std::cout;

int main()
{
    double A, B, C;
    std::cin >> A >> B >> C;

    double delta = B * B - 4 * A * C;

    cout << delta << '\n';

    if(A == 0 || delta < 0)
    {
        cout << "Impossivel calcular\n";
    } 
    else
    {
        double r1 = ((B * -1) + std::sqrt(delta)) / (2 * A);
        double r2 = ((B * -1) - std::sqrt(delta)) / (2 * A);

        cout << std::fixed << std::setprecision(5);

        cout << "R1 = " << r1 << '\n';
        cout << "R2 = " << r2 << '\n';
    }

    return 0;
}

File: iniciante/1036/1036.py
# 1036 - Fórmula de Bhaskara - beecrowd
# Iniciante - Nível 3

A, B, C = map(float, input().split())

delta = B ** 2 - 4 * A * C

if A == 0 or delta < 0:
    print("Impossivel calcular")
else:
    r1 = ((B * -1) + delta ** (1/2)) / (2 * A)
    r2 = ((B * -1) - delta ** (1/2)) / (2 * A)

    print(f"R1 = {r1:.5f}")
    print(f"R2 = {r2:.5f}")
